src/main/java/Main.java

```diff
@@ -1,8 +1,26 @@
-public class Main {
-
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
 
+public class Main {
     public static void main(String[] args) {
-
+        TreeMap<Integer, Integer> numbers;
+        try {
+            numbers = new TreeMap<>();
```

Comment:
Jeżeli wszystko jest w try, to nie ma potrzeby deklarować mapy wcześniej, tylko można to połączyć w 1 wiersz